fixtures/schedule/index.html

```diff
@@ -464,6 +473,22 @@ <h2>Tests:</h2>
     console.log('scheduled cbE');
   };
 }
+
+function runTestSeven() {
+  // Test 7
+  // Calls callbacks, continues calling them even when this tab is in the
+  // background
+  clearTestResult(7);
+  let counter = -1;
+  function incrementCounterAndScheduleNextCallback() {
+    const counterNode = document.getElementById('test-7');
+    counter++;
+    counterNode.innerHTML = counter;
+    waitForTimeToPass(100);
```

Comment:
Why 100?

Reply:
Good question~! It's arbitrary - matches what the comparable wrappers were doing internally at Facebook.

Reply:
Cool. I didn't know about the scheduling fixtures! This is really neat!